doc/whatsnew/2/2.15/index.rst

```diff
@@ -18,3 +18,163 @@ We improved ``pylint``'s handling of namespace packages. More packages should
 be linted without resorting to using the ``--recursive=y`` option.
 
 .. towncrier release notes start
+
+What's new in Pylint 2.15.0?
+----------------------------
+Release date: 2022-08-25
+
```

Comment:
Do we want to add a little summary?

Reply:
There's a small summary from the "alpha" we did not realease from line 14 to line 18. I don't remember any major thing to add, I'm tempted to add something about a "miscellaneous bug hunting session", what do you think ?

Reply:
@mbyrnepr2 became part of the team so there's that :)

Reply:
Perhaps one final comment about the documentation project? It died down recently and with new influx of college students in September this might be a good time to do one final advertisement for it?

Reply:
Done in https://github.com/PyCQA/pylint/pull/7355/commits/f958f24b75500f341866a1440643dfecaa6b5e56